FloatingTextField/AppDelegate.swift

```diff
@@ -12,35 +12,4 @@ import UIKit
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
-
```

Comment:
There were several whitespace violations here, but I'd rather remove these methods altogether. Since we are not using them, they're only adding clutter.

Always keep your codebase as clean as possible.